american-truck-simulatorupdates.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "UpdateStageName":"SteamCMD Download",
+        "UpdateSourceData":"2239530",
```

Comment:
Please include UpdateSourceArgs with client ID for completeness

Reply:
Thanks for calling that out. Need to add that to the config generator as well.